src/extension.ts

```diff
@@ -201,6 +203,9 @@ export function activate(context: vscode.ExtensionContext) {
 	let startIntegration = vscode.commands.registerCommand('camelk.startintegration', async (uri:vscode.Uri) => { await runTheFile(uri);});
 	context.subscriptions.push(startIntegration);
 
+	// add commands to create config-map and secret objects from .properties files
```

Comment:
not sure this comment useful.
renaming the class and the method should be enough to make it clear.

Reply:
Leaving the comment. I think it's provides additional context without a lot of overhead (a single comment line).